src/ape/managers/chain.py

```diff
@@ -359,9 +360,22 @@ def base_fee(self) -> int:
     def pending_timestamp(self) -> int:
         """
         The current epoch time of the chain, as an ``int``.
+        Set the timestamp to fast-forward time into the future.
+
+        Usage example::
+
+            from ape import chain
+
+            chain.pending_timestamp += 3600
         """
 
-        return int(time.time())
+        return int(time.time() + self._time_offset)
+
+    @pending_timestamp.setter
+    def pending_timestamp(self, new_value: int):
+        provider = self._get_test_provider(TestProviderAPI.fast_forward_time.__name__)
+        new_timestamp = provider.fast_forward_time(new_value)
```

Comment:
When you do `chain.pending_timestamp += 3600`, it first add the current pending timestamp (which is the OS time + the existing offset which starts at `0`). Then, it calls `time_travel()` from `eth-tester` with the new timestamp (or the evm method in hardhat, not implemented yet)